### DISCUSSION TITLE
Space with margin is behaving differently

### DISCUSSION THREAD

**Discussion by zarub2k** [tailwindlabs/tailwindcss#6558] [Help]
I have a parent container <div> with space-y-8 defined. When I want to give some special gap for any of the children I specifically added mt-8 to the child. But this is not getting effect. The moment I removed the space-y-8 this is getting changed.

Is there a possibility to use both space and margin utilities? Or is this a bug?

**Comment by NovusOpus** [comment]
If you check the [Space Between Class list](https://tailwindcss.com/docs/space#add-vertical-space-between-children) and scroll down the list  until you find `space-y-8`.

Then you'll see the CSS Value is: `margin-top: 2rem;`.

So For example let's say you got something like this:
```html
<div class="min-h-screen grid place-items-center">
  <div class="bg-blue-900 w-96 h-[100%] flex flex-col space-y-8 justify-center items-center">
    <div class="w-48 bg-white/50 text-center">
      1 Child
    </div>
    <div class="w-48 bg-white/50 text-center">
      2 Child
    </div>
    <div class="w-48 bg-white/50 text-center">
      3 Child
    </div>
  </div>
</div>
```
So when you use `space-y-8`, you setting `margin-top: 2rem;` to every child element except for the first one.  

And when you add `mt-8` to any child except for the first one, the CSS Value is: `margin-top: 2rem;`. But since you applying the same CSS Property Twice your CSS equivalent is going to look like this:

```postcss
child {
  margin-top: 2rem;
  margin-top: 2rem;
}
```
So your CSS is not Valid, that's why, when you use mt-8 on child 2 or 3, it's going to be ignored. If you use it on the first child it's going to work since the first child it's not getting affected by the `margin-top: 2rem;` from `space-y-8`.
